test/parsers/rss2_test.exs

```diff
@@ -7,16 +7,18 @@ defmodule Feedme.Test.Parsers.RSS2 do
   setup do
     sample1 = XmlNode.from_file("test/fixtures/rss2/sample1.xml")
     sample2 = XmlNode.from_file("test/fixtures/rss2/sample2.xml")
+    sample3 = XmlNode.from_file("test/fixtures/rss2/cre.xml")
```

Comment:
can you please rename sample name to more descriptive since it's not a standard rss2 feed.